core.fnl

```diff
@@ -211,3 +215,14 @@ Returns nil. This function causes side-effects.
                     (let [module (require path)]
                       {path (module.init config)})))
              (reduce #(merge $1 $2) {})))
+
+
+(fn docstr-f [x y z]
+  "The docstr-f docstr"
+  (print "hi"))
+
+(fn no-fun [x y z]
```

Comment:
What is this and `docstr-f` for?

Reply:
Oh! Should have deleted that. 

Had to play with some fennel config settings to test copying the docstr on the advised function wrapper